.github/workflows/create-tag.yaml

```diff
@@ -0,0 +1,42 @@
+name: Tag
```

Comment:
Maybe we don't need to backport this one? Github should just use the one in master, right?

Reply:
we need this to be backported because, when creating a tag via UI there is a branch name dropdown menu and we want to create the tag from a particular release branch so that our action release/tag will fetch the branch name.

Reply:
It doesn't hurt to backport, but it still doesn't seem necessary. All the release branches already show in the drop down, and the selection will be passed to the action.

Reply:
It is correct that all the release branches will already show in the drop-down but suppose, if I have this action merged in release-1.7 only and we try to run this from the release-1.6 branch it will show not allow that with this message
 ```Workflow does not exist or does not have a workflow_dispatch trigger in this branch.```
I tried  in my fork and says this only
![Screenshot from 2021-08-17 19-28-57](https://user-images.githubusercontent.com/34444611/129739514-bf9ece61-0c0c-4858-b788-2fc485c26abf.png)